tests/python/pants_test/pantsd/test_pantsd_integration.py

```diff
@@ -543,8 +543,8 @@ def test_signal_pailgun_stream_timeout(self):
 Interrupted by user over pailgun client!
 """
                .format(today=re.escape(today))],
-      quit_timeout=0.01,
-    )
+      # NB: Make the timeout very small to ensure the warning message will reliably occur in CI!
+      quit_timeout=0.000001)
```

Comment:
how about `1e-6` (just a little easier to read)

Reply:
I mostly just pressed 0 until it seemed small enough, fixed! Also fixed the error message which uses a similar form.